py-polars/polars/dataframe/group_by.py

```diff
@@ -448,30 +448,44 @@ def all(self) -> DataFrame:
         """
         return self.agg(F.all())
 
-    def len(self) -> DataFrame:
+    def len(self, name: str | None = None) -> DataFrame:
```

Comment:
This should also be added to the LazyGroupBy variant.

Reply:
Good catch; done 👌